packages/Webkul/Admin/src/Resources/views/layouts/nav-aside.blade.php

```diff
@@ -1,7 +1,7 @@
 <div class="aside-nav">
 
     {{-- button for collapsing aside nav --}}
-    <nav-slide-button icon-class="accordian-left-icon"></nav-slide-button>
+    <nav-slide-button  @if (core()->getCurrentLocale() && core()->getCurrentLocale()->direction == 'rtl') icon-class="accordian-right-icon" :direction="'rtl'" @else icon-class="accordian-left-icon" @endif></nav-slide-button>
```

Comment:
When language changes body tag always have `.rtl` class, you just need to write the classes in the `packages/Webkul/Velocity/src/Resources/assets/sass/components/rtl.scss`.